taxonomy.php

```diff
@@ -16,20 +16,40 @@
 
 $context              = Timber::get_context();
 $context['page_type'] = get_queried_object();
-$pagetype_posts       = Timber::get_posts();
 
-if ( $pagetype_posts ) {
-	foreach ( $pagetype_posts as $pagetype_post ) {
-
-		$author_override                = get_post_meta( $pagetype_post->ID, 'p4_author_override', true );
-		$pagetype_post->author          = '' === $author_override ? get_the_author_meta( 'display_name', $pagetype_post->post_author ) : $author_override;
-		$pagetype_post->author_url      = '' === $author_override ? get_author_posts_url( $pagetype_post->post_author ) : '#';
-		$pagetype_post->author_override = $author_override;
-	}
+function setAuthorOverride($post) {
+	$author_override       = get_post_meta( $post->ID, 'p4_author_override', true );
+	$post->author          = '' === $author_override ? get_the_author_meta( 'display_name', $post->post_author ) : $author_override;
+	$post->author_url      = '' === $author_override ? get_author_posts_url( $post->post_author ) : '#';
+	$post->author_override = $author_override;
+	return $post;
 }
 
-$context['posts'] = $pagetype_posts;
-
 $context['wp_title'] = $context['page_type']->name;
 
-Timber::render( $templates, $context );
+wp_register_script( 'load_more', get_template_directory_uri() . '/assets/js/load_more.js', [ 'jquery', 'main' ], '0.0.1', true );
+wp_enqueue_script( 'load_more' );
+
+$post_args = [
+	'posts_per_page' => 10,
+	'post_type'      => 'post',
+	'paged'          => 1,
+];
+
+if ( get_query_var( 'page' ) ) {
+	$templates          = [ 'tease-page-type.twig' ];
+	$post_args['paged'] = get_query_var( 'page' );
+	$pagetype_posts     = new \Timber\PostQuery( $post_args );
```

Comment:
Then you can use the functions making the posts P4_Post objects with this
`$pagetype_posts     = new \Timber\PostQuery( $post_args, 'P4_Post' );`

and making any changes needed in the twig template